### DISCUSSION TITLE
Error on entering worker: `find: ‘./proc/12/map_files’: Permission denied`

### DISCUSSION THREAD

**Discussion by benwhalley** [dokku/dokku#4952] [Q&A]
I'm not sure what trigged this, but when I try to enter my worker (or web) processes now for any app in the instance I get the following error:

`find: ‘./proc/12/map_files’: Permission denied`

(Different apps have different process numbers, but the pattern is the same).

Does anyone have any suggestions for trouble shooting this? AFAICT nothing changes on the system apart from (maybe) automatic apt- security upgrades.

**Comment by josegonzalez** [comment]
Can you run paste the command you ran to enter the processes? Also, what version of dokku are you running? Finally, can you run that command with trace mode on? For instance:

```
dokku --trace enter $APP web
```

And gist the output?

**Reply by benwhalley** [reply]
Hey - thanks for this. Sorry for the delay - didn't get pinged that this was here.

The gist of the output is here: https://gist.github.com/benwhalley/9e425f268de4cc3fcc540ddcdbfe739b

I am on: `dokku version 0.19.6`
